src/dataMigration.js

```diff
@@ -235,6 +236,17 @@ const dataMigrations = [
       });
     },
   },
+  // 3.2.0 Adds indicator and dashboard functionality. This functionality requires certain tables
+  // to be synced to mobile. Sending a message to the server that the mobile site has upgraded
+  // will trigger syncing of the needed tables.
+  {
+    version: '3.2.0-rc1',
+    migrate: (database, _, fromVersion, toVersion) => {
+      database.write(() => {
+        createRecord(database, 'Message', fromVersion, toVersion);
```

Comment:
can we relocate this around line 51, around `AsyncStorage.setItem(APP_VERSION_KEY, toVersion);`, if toVersion !== fromVersion` ?

Reply:
nice 
0833d2e